docker/base

```diff
@@ -1,6 +1,9 @@
 #!/bin/sh
 set -xeu
 
+# Get utility stuff
+. ./util.sh
```

Comment:
Minor nitpick: is there any semantical difference between `. ./util.sh` and `source ./util.sh`? I usually prefer former as more obvious.